views/index.erb

```diff
@@ -0,0 +1,33 @@
+
+
+<header class="page-header">
+  <h1>Chitter </h1>
+</header>
+
+<img class="fit-picture"
+     src="https://i.ytimg.com/vi/TTe-4vqT-10/maxresdefault.jpg"
+     alt="Grapefruit slice atop a pile of other slices">
+
+
+<main>
+    <p> Welcome to  <em>Chitter</em> !</p>
+</main>
+
+
+<dl>
+  <% @messages.reverse_each do |message|%>
+    <dt><strong><%= message.name %></strong>  </dt>
+    <dd> <em>PEEPS</em> <%= message.message %> <%=Time.now%>  </dd>
+    
+  <% end %>
+</dl> 
+
+
+
+<form action "/messages" method = "post" >
+<input type="text" name="message" placeholder ="What is  happening?">
+<input type="text" name="name" placeholder ="name">
+<input type="submit" value="peep">
```

Comment:
I like the peep form being on the main page, more elegant than moving to a separate page. :+1: